StartHere.md

```diff
@@ -20,22 +20,17 @@ Discuss issues you've faced, discover new & better ways of working (together), c
 
 <a href="https://www.meetup.com/devstaff/"><img src="images/meetup.png" align="left"  height="80" ></a>
 
-The main tool for organizing our meetings is *meetup.com*. The first step is head [there](https://www.meetup.com/devstaff/) and join the group. 
+The main tool for organizing our meetings is *meetup.com*. The first step is to head [there](https://www.meetup.com/devstaff/) and join the group. 
 
-Make sure you configure your notification options on meetup.com to receive notifications about upcoming events!
+Make sure you configure your notification options on meetup.com in order to receive notifications about upcoming events!
```

Comment:
We could mention here that we use meetup notifications for other activities and news of the community apart from the upcoming meetups.